frontend/src/pages/settings/account/Account.tsx

```diff
@@ -7,20 +7,55 @@ import ContentHeader from '../../../components/ContentHeader';
 import PasswordForm from '../../../features/settings/account/PasswordForm';
 import DeleteAccountForm from '../../../features/settings/account/DeleteAccountForm';
 import SettingsMenu from '../../../features/settings/SettingsMenu';
+import { EmailAddressForm } from 'src/features/settings/account/EmailAddressForm';
 
 const useStyles = makeStyles((theme) => ({
   titleDanger: { color: theme.palette.error.main },
 }));
 
-function AccountPage() {
-  const classes = useStyles();
+const Section = ({
+  title,
+  children,
+  ...rest
+}: {
+  title: string | React.ReactNode;
+  children: React.ReactNode;
+  [rest: string]: unknown;
+}) => {
+  let sectionTitle;
+  if (typeof title === 'string') {
+    sectionTitle = (
+      <Typography variant="h4" color="secondary">
+        {title}
+      </Typography>
+    );
+  } else {
+    sectionTitle = title;
+  }
```

Comment:
I prefer the following:

```suggestion
  const sectionTitle = (typeof title === 'string') 
    ?  (<Typography variant="h4" color="secondary">
        {title}
      </Typography>) 
    : title
```

The two advantages I find are 1. more concise and 2. using `const` instead of `let` reassures me that I don't need to read more code to know what `sectionTitle` is.